pkg/controller/ibmblockcsi/ibmblockcsi_controller.go

```diff
@@ -56,7 +56,14 @@ import (
 )
 
 // ReconcileTime is the delay between reconciliations
-const ReconcileTime = 30 * time.Second
+const (
+	ReconcileTime = 30 * time.Second
+	Node = "node"
+	Controller = "controller"
+)
+
+var daemonSet_restarted_key = ""
```

Comment:
this can be a `fmt.Sprintf("%s/restartedAt", oconfig.APIGroup)` "const" here

Reply:
I didn't do it because I wanted that as long as the `restartedAt` annotation won't be on the `daemonset` it will stay empty, that the sycer will know that there isn't a new annotation that it needs to take care of

Reply:
it can check the value, I don't think the key is interesting

Reply:
oh, you don't want `reconcileServiceAccount` to return anything...

Reply:
`reconcileServiceAccount` return only error, because `daemonSetRestartedKey` and `daemonSetRestartedValue` are global variables

Reply:
so I can check the value and move the to the const section if you like

Reply:
I will try to find a better way

Reply:
could you open a ticket for it and write its number here?
I guess it won't be for this release

Reply:
done